querydsl-jpa/src/test/java/com/querydsl/jpa/JPABase.java

```diff
@@ -273,10 +270,9 @@ public void createQuery2() {
 
     @Test
     public void createQuery3() {
```

Comment:
Not for this review but poor naming. What's the test doing? createQuery3? Runs a query, gets a list, checks the entries are not null?

Reply:
Doing parametrized tests in JUnit4 is not that easy so that looks like a simulation of that to me

Reply:
There are a lot of these scattered across the different suites. They don't seem to be just parameterised. In some cases they check different functionality.